docs/user/dql/functions.rst

```diff
@@ -1425,6 +1425,54 @@ Example::
     | 26                               |
     +----------------------------------+
 
+DAY_OF_MONTH
+----------
+
+Description
+>>>>>>>>>>>
+
+Usage: day_of_month(date) extracts the day of the month for date, in the range 1 to 31. The dates with value 0 such as '0000-00-00' or '2008-00-00' are invalid.
+
+Argument type: STRING/DATE/DATETIME/TIMESTAMP
+
+Return type: INTEGER
+
+Synonyms: `DAY`_, `DAYOFMONTH`_
+
+Example::
+
+    os> SELECT DAY_OF_MONTH('2020-08-26')
+    fetched rows / total rows = 1/1
+    +------------------------------+
+    | DAY_OF_MONTH('2020-08-26')   |
+    |------------------------------|
+    | 26                           |
+    +------------------------------+
+
+    os> SELECT DAY_OF_MONTH(DATE('2020-08-26'))
```

Comment:
all of these similar tests are covered by IT tests.  We can just have the one example in the documentation.

Reply:
Fixed in 878a6a152955d3cbbc1a21c85ff2d900a6bc785c